deploy-manage/deploy/self-managed/install-elasticsearch-with-debian-package.md

```diff
@@ -135,17 +145,29 @@ sudo dpkg -i elasticsearch-{{stack-version}}-amd64.deb
 
 ### Security at startup [deb-security-configuration]
 
-:::{include} _snippets/auto-security-config.md
+:::{include} _snippets/auto-security-config-rpm-deb.md
+:::
+
+## Step 6: Reset the `elastic` superuser password
+
+:::{include} _snippets/reset-superuser-rpm-deb.md
 :::
 
 :::{include} _snippets/pw-env-var.md
 :::
 
-## Step 6: Check that {{es}} is running [deb-check-running]
+## Step 7: Check that {{es}} is running [deb-check-running]
 
 :::{include} _snippets/check-es-running.md
 :::
 
+## Step 8 (Multi-node clusters only): Clean up the config files
```

Comment:
Not my expertise area, but would `consolidate configuration files` sound better than cleaning up?  For your review and consideration.

Reply:
`consolidate` to me means that you are merging multiple config files into fewer config files, instead of editing them

Reply:
changed to "update"